src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerWeb.tsx

```diff
@@ -77,12 +85,55 @@ export function VideoEmbedInnerWeb({
     }
   }, [embed.playlist])
 
+  // purge low quality segments from buffer on next frag change
+  useEffect(() => {
+    if (!hlsRef.current) return
+
+    const current = hlsRef.current
+
+    if (focused) {
+      function fragChanged(
+        _event: Events.FRAG_CHANGED,
+        {frag}: FragChangedData,
+      ) {
+        if (!hlsRef.current) return
```

Comment:
i mean here, can you use the captured variable instead?

Reply:
can do, I've never been sure what the best way to go about it is